docs/contributing/05-gh-actions.md

```diff
@@ -0,0 +1,206 @@
+
+**Github Actions Security Best Practices - Cheat Sheet from GitGuardian**
+![Github Actions Security Best Practices - Cheat Sheet](assets/GitHub-Actions-Security-Best-Practices_cheatsheet-1.png)
+
+## Appendix A: Approved Github Actions
+
+### golangci/golangci-lint-action
+
+This action runs the [golangci-lint](https://golangci-lint.run/) tool from GolangCI.
+
+- Repo: <https://github.com/golangci/golangci-lint-action>
+- Marketplace: <https://github.com/marketplace/actions/run-golangci-lint>
+- Repo `golangci-lint` tool: <https://github.com/golangci/golangci-lint>
+
+#### Rating
+
+- openSSF Scorecard rating (15.08.2023)
+  - [golangci-lint-action](./scorecard-ratings/golangci_golangci-lint-action.json): 7.2/10
+  - [golangci-lint](./scorecard-ratings/golangci_golangci-lint.json): 6.3/10
+
+The tools (the action as well as the tool itself) are very popular and a large community is maintaining both.
+
+#### Versions
+
+- golangci/golangci-lint-action@639cd343e1d3b897ff35927a75193d57cfcba299
+- golangci/golangci-lint-action@v3
+
+### AbsaOSS/k3d-action
+
+- Repo: <https://github.com/AbsaOSS/k3d-action>
+- Marketplace: <https://github.com/marketplace/actions/absaoss-k3d-action>
+
+The action contains a shell scripts which installs a K3D cluster on the runner. The [URL of the installation script of K3D](https://github.com/AbsaOSS/k3d-action/blob/4e8b3239042be1dc0aed6c5eb80c13b18200fc79/run.sh#L27) is hardcoded in the script.
+
+#### Rating
+
+- openSSF Scorecard rating (16.08.2023)
+  - [AbsaOSS/k3d-action](./scorecard-ratings/AbsaOSS_k3d-action.json): 4.8/10
+
+#### Versions
+
+- AbsaOSS/k3d-action@4e8b3239042be1dc0aed6c5eb80c13b18200fc79
+
+### zgosalvez/github-actions-ensure-sha-pinned-actions
+
+- Repo: <https://github.com/zgosalvez/github-actions-ensure-sha-pinned-actions>
+- Marketplace: <https://github.com/marketplace/actions/ensure-sha-pinned-actions>
+
+#### Rating
+
+- openSSF Scorecard rating (16.08.2023)
+  - [zgosalvez/github-actions-ensure-sha-pinned-actions](./scorecard-ratings/zgosalvez_github-actions-ensure-sha-pinned-actions.json): 5.1/10
+
+Einschätzung von Johannes Vogt:
+> Hi Torsten, hab's mir angeschaut und der Code sieht vernünftig aus.
+>
+> Es werden alle ya?ml Dateien durchgegangen und geschaut, ob in irgendeinem unter dem Pfad jobs[...].uses oder jobs[...].steps[x].uses die sha Values die richtige Länge haben (40 für sha1 und 64 für sha256)... Mit einer allowlist kann man den Check für bestimmte Actions ignorieren.
+>
+> Der Code wird mittels <https://github.com/vercel/ncc> in eine einzelne Datei kompiliert. Den Output hab ich mir nicht genauer angeschaut
```

Comment:
What is the purpose of this passage? Why is it in German?

Reply:
This passage is the assessment result of an experienced JavaScript developer about the GitHub Action. I added it in German to keep the original content without changes because of the translation. Tbh, I want to keep it because it is a quote but I agree that the title should be in English and the purpose should be explained better. If it is okay for you, I will do this with the next release of this file (which comes soon).

Reply:
Maybe for the time being you could only translate the phrase `Einschätzung von Johannes Vogt:` to `Assessment by Johannes Vogt:`. WDYT?

Reply:
@IwonaLanger Agree. Can you propose a change and I will accept it?